Wiktionary/WiktionPage/spec/irregular_conjug.nlp

```diff
@@ -0,0 +1,63 @@
+###############################################
+# FILE: irregular_conjug
+# SUBJ: comment
+# AUTH: Eduardo Costa
+# CREATED: 2024-1-19 20:24:40
+# MODIFIED:
+###############################################
+
+@PATH _ROOT _textZone _headerZone _headerZone _conjugZone _LINE
+
+@POST
+G("verb_form_cnt") = ++G("verb_form_cnt");
+
+L("form_cnt") = G("verb_form_cnt");
+L("verb_form") = strclean(N("$text",2) + " " + N("$text",4));
+
+G("verb_form")[L("form_cnt")] = L("verb_form");
```

Comment:
Eu comecei o array aqui da segunda posição para facilitar a correlação entre a posição no array e as posições usadas pela macro mas se for necessário posso mudar isso para não desperdiçar 1 posição no array